Prog/vacant-lots-fairfax.Rmd

```diff
@@ -664,28 +627,143 @@ There are `r ` observations that did not have a match in the zoning code. Below,
 
 ```{r antijoin}
 
-anti_join(vacantfinal, zoningcode, by = c("zoning_code"="code")) %>% 
+anti_join(jurfinal, zoningcode, by = c("zoning_code"="code")) %>% 
   count(zoning_code) %>% arrange(desc(n)) %>%
   knitr::kable()
 
 ```
 
-It is unclear why there are so many codes in the blackknight data that are not readily identifiable in the zoning code. 
+
+Categorize those lots that are not joining
+```{r unjoined-categories}
+planned<-c("PRC",
+           "PDH-3",
+           "PDH-4",
+           "PDH-8",
+           "PDH-2",
+           "PDH-12",
+           "PDC",
+           "PDH-16",
+           "PDH-5",
+           "PDH-20",
+           "PRM",
+           "PDH-30",
+           "PDH-1",
+           "PDH-40",
+           "PDH",
+           "PLANNED RES MIXED USE",
+           "PRM",
+           "PTC")
+
+notResidential <- c("I-5",
+                    "C-2",
+                    "C-8",
+                    "C-6",
+                    "C-3",
+                    "I-4",
+                    "C-5",
+                    "I-3",
+                    "COMM ",
+                    "I-6",
+                    "INDUSTRIAL ",
+                    "C-7",
+                    "INDUSTR ",
+                    "PUBLIC SERVICE CORPORATIO",
+                    "C-1",
+                    "C-4",
+                    "CD",
+                    "I-I",
+                    "I-2",
+                    "CO",
+                    "CS",
+                    "COM ",
+                    "ID",
+                    "CM",
+                    "CMP",
+                    "C-1A",
+                    "CCD",
+                    "I-1",
+                    "INDUST W/OTHER INDUST ZON"
+)
+  
+residential_weird <- c("RES ",
```

Comment:
how many vacant lots are we dropping because they are "weird" - also we are getting "weird" codes from the join to the zoning layer?

Reply:
35 are vacant - the merge with the zoning layer is not working for those classified as "residential weird" because these data also have missing lat-lons. They have BlackKnight zoning info but that's it.